src/test/java/com/microsoft/jenkins/kubernetes/integration/integration-test.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+
+show_help() {
+    cat <<EOF
+In the project root directory, run the following command and pass in the service principal details in parameters.
+
+    bash src/test/java/com/microsoft/jenkins/kubernetes/integration/integration-test.sh
+
+    Prerequisites:
+        * Logged in Azure CLI
```

Comment:
Since this plugin is a general k8s plugin, provide an option that user can run the test without Azure (suppose they have already have a k8s cluster)?

Reply:
If the user pass in all the details, we will skip the Azure ACS & ACR setup. We have a check to ensure that

```bash
resource_group=
if [[ -z "$KUBERNETES_CD_MASTER_HOST" ]] || [[ -z "$KUBERNETES_CD_DOCKER_REPOSITORY" ]]; then
    resource_group="k8s-test-$suffix"
fi
```

However, currently the check for `az account` happens too early. It is not necessary if user do not need to provision Azure resources. I will update the code logic and the documentation.